app.R

```diff
@@ -0,0 +1,1382 @@
+APP_TITLE <<- "Log_Transformation"
+APP_DESCP  <<- paste(
+  "This app provides the information when to use log transformations", 
+  "to linearize data, how to analyze scenarios where a Log Transform", 
+  "is needed and when it is not."
+)
+## End App Meta Data------------------------------------------------------------
+
+# Define the UI ----
+ui <- list(
+  dashboardPage(
+    skin = "green",  
+    ### Create the app header
+    dashboardHeader(
+      titleWidth = 250, 
+      title = "Log Transformations", 
+      tags$li(class="dropdown", 
+              actionLink("info", icon("info"), class="myClass")), 
+      tags$li(
+        class = "dropdown", 
+        tags$a(target = "_blank", icon("comments"), 
+               href = "https://pennstate.qualtrics.com/jfe/form/SV_7TLIkFtJEJ7fEPz?appName=Log_Transformations"
+        )
+      ), 
+      tags$li(class='dropdown', 
+              tags$a(href="https://shinyapps.science.psu.edu/", 
+                     icon('home', lib='font-awesome')))), 
+    dashboardSidebar(
+      width = 250, 
+      sidebarMenu(
+        id = "pages", 
+        menuItem("Overview", tabName = "overview", icon = icon("tachometer-alt")), 
+        menuItem("Explore", tabName = "explore", icon = icon("wpexplorer")), 
+        menuItem("References", tabName = "References", icon = icon("leanpub"))
+      ), 
+      tags$div(
+        class = "sidebar-logo", 
+        boastUtils::psu_eberly_logo("reversed")
+      )
+    ), 
+    dashboardBody(
+      tabItems(
+      tabItem(tabName = "overview", 
+              h1("Log Transformation"), 
+              p("The goals of this app are to know when to use log 
+              transformations to linearize data, how to analyze scenarios
+                where a Log Transform is needed and when it is not."), 
+              p("There are three different datasets, which are about Animals, 
+                Earthquakes, and Countries."), 
+              br(), 
+              h2("Instructions"), 
+              tags$ol(
+                tags$li("In the application, you will see a number of 
+                        different data sets you can go through."),  
+                tags$li("Go through each data set and look through 
+                the different variables to check whether
+                        a log transformation would help."),  
+                tags$li("Use the 'Show Histogram' button to see each 
+                        individual variable more clearly."),  
+                tags$li("You can also examine log transforms on 
+                        variables in your own data set!")
+              ), 
+              div(
+                style = "text-align: center", 
+                bsButton(
+                  inputId = "go1", 
+                  label = "GO!", 
+                  size = "large", 
+                  icon = icon("bolt"), 
+                )
+              ), 
+              br(), 
+              br(), 
+              h2("Acknowledgement"), 
```

Comment:
Make sure to add your name to the acknowledgements as well as the updated by statement